modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -14,6 +14,19 @@ set(TFM_VALID_PARTITIONS
   TFM_PARTITION_AUDIT_LOG
   )
 
+# List of all crypto modules that can be enabled/disabled
+# Name must match start of name found in 'trusted-firmware-m/config/config_default.cmake'
```

Comment:
Nit: Specify which names in config_default.cmake:
```suggestion
# Corresponds to the *_MODULE_DISABLED configs in 'trusted-firmware-m/config/config_default.cmake'
```

Reply:
Sure, can do that.

Reply:
done @oyvindronningstad, thanks.